package.json

```diff
@@ -221,7 +220,6 @@
     "@react-native/typescript-config": "^0.74.1",
     "@testing-library/jest-native": "^5.4.1",
     "@testing-library/react-native": "^11.5.2",
-    "@tsconfig/react-native": "^2.0.3",
```

Comment:
what was this for?

Reply:
The tsconfig.json was extended from `@tsconfig/react-native` in https://github.com/bluesky-social/social-app/commit/d6ec627c8cd32836e5ed494606318959ca17fca1#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0

It was changed to `@react-native/typescript-config` in https://github.com/bluesky-social/social-app/commit/93f5bb3b1f10cb6011f1770132067fc58df39f3b#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R2